src/db/scripts/grantPermissions.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+
+// Should be called with two options: email, role.
+// Role is one from constants/permissions.
```

Comment:
@UladBohdan please add here example of call or, better, `console.log` for calling without / with bad params